index.html

```diff
@@ -169,10 +169,12 @@ <h2 class="color-secondary">About me</h2>
     <section class="contact">
         <div class="email">
             <h4 id="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in</h4>
-            <input class="color-primary" type="text" placeholder="Full name" required>
-            <input class="color-primary" type="email" placeholder="Email address" required>
-            <textarea placeholder="Enter text here" name="comment" cols="30" rows="10" required></textarea>
-            <button class="get-intouch-btn color-primary">Get in touch</button>
+            <form action="https://formspree.io/f/xpzewrow" method="post">
+                <input class="color-primary" type="text" placeholder="Full name" required maxlength="30">
+                <input class="color-primary" type="email" placeholder="Email address" required>
+                <textarea name="comment" cols="30" rows="10" placeholder="Enter text here" required maxlength="500"></textarea>
+                <button class="get-intouch-btn color-primary button">Get in touch</button>
```

Comment:
Although the form is getting submitted when the button is clicked, but as a best practice it is advisable to add the attribute of `type="submit"` to the button, this will make your code more understandable and make it clear to a newbie what this button is for.